src/main/java/net/maritimeconnectivity/serviceregistry/models/dto/mcp/McpCertitifateDto.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2021 GLA Research and Development Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package net.maritimeconnectivity.serviceregistry.models.dto.mcp;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import net.maritimeconnectivity.serviceregistry.utils.LocalDateTimeDeserializer;
+import org.grad.secom.core.base.DateTimeSerializer;
+
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+
+/**
+ * The MCP Certificate DTO Class.
+ * <p/>
+ * This class is used to load the represent the certificate entries as they
+ * come directly from the MCP Identity Registry.
+ *
+ * @author Nikolaos Vastardis (email: Nikolaos.Vastardis@gla-rad.org)
+ */
+public class McpCertitifateDto {
```

Comment:
Just noticed that `Certificate` is misspelled here
```suggestion
public class McpCertificateDto {
```

Reply:
Think all the occurrences need to renamed as well.